docs/roadmap.md

```diff
@@ -0,0 +1,18 @@
+| Q4 2023     | FastAPI Refactor <br><br> LLM-powered search                                                                                      | :white_check_mark: <br><br> :white_check_mark:                     | Refactored to an all-components-in-one-repo codebase <br> Match user questions to content in the database using embeddings from LLMs (Open AI)                                                                                                                 |
+| **Q1 2024** | LLM responses <br> <br> AI Guardrails <br> <br> Multi-lingual content                                                             | :white_check_mark: <br><br> :white_check_mark: <br><br> :tools:    | Craft a custom response to the question using LLM chat and the content in the database <br> Keep LLM responses friendly and strictly context-based <br> Support storage and matching of multi-language variants of FAQs                                        |
+| Q2 2024     | Support for low resource language <br><br> Admin app 2.0 <br><br> Intent Classification & Message Triaging                        | :construction: <br><br><br> :construction: <br><br> :construction: | Support question-answering in regional languages. First targets: Xhosa, Zulu, Hindi, Igbo, Hausa <br> A new and better look for Admin App that supports multilingual content management <br> Tag user messages with intents & flag urgency                     |
+| Q3 2024     | Content Management Dashboard & Engineering Dashboard <br><br><br> Multi-turn chat <br> <br><br> Document-based question answering | :pencil: <br><br><br><br><br>:pencil: <br><br><br>:pencil:         | Monitor uptime, response rates, throughput HTTP response codes. See content use, questions that receive poor feedback, missing content, and more. <br><br> Refine or clarify user question through conversation. <br><br> Retrieve answers from a document set |
+| Q4 2024     | Voice notes support <br><br><br> Multimedia content <br><br><br> A/B Testing                                                      | :pencil: <br><br><br>:pencil: <br><br><br>:pencil:                 | Automatic Speech Recognition for audio message to content matching <br><br> Respond with not just text but images and audio as well. <br> <br> Test and decide content that works better for users                                                             |
+
+!!! info "Key <br> :white_check_mark: - Completed <br> :tools: - In active development <br> :construction: - Queued <br>:pencil: - Yet to be scoped"
+
+On The Horizon
```

Comment:
How should I think about "On the Horizon" vs. the ones above?

Reply:
Thanks @sidravi1, will make it `Beyond 2024` then